src/cli/commands/deploy/deploy.ts

```diff
@@ -34,6 +34,11 @@ export async function deploy(options: SWACLIConfig) {
   // make sure appLocation is set
   appLocation = path.resolve(appLocation || process.cwd());
 
+  // make sure dataApiLocation is set
+  if(dataApiLocation) {
+    dataApiLocation = path.resolve(dataApiLocation);
```

Comment:
should we be logging in the console about the dataApiLocation

Reply:
Good catch thanks Sulabh, this & error logs/validation of path is missing, adding it